src/main/daml/Daml/Finance/Settlement/Factory.daml

```diff
@@ -30,19 +30,22 @@ template Factory
       instruct SettlementFactory.Instruct {instructors; settlers; id; description; contextId;
         routedSteps; settlementTime} = do
           let
-            createInstruction routedStep index = Instruction with
-              requestors = instructors
-              settlers
-              batchId = id
-              id = Id (show index)
-              routedStep
-              settlementTime
-              allocation = Unallocated
-              approval = Unapproved
-              signedSenders = empty
-              signedReceivers = empty
-              observers = M.fromList [(show id, settlers)]
-            instructions = mapWithIndex createInstruction routedSteps
+            createInstruction index routedStep =
+              ( index + 1,
+                Instruction with
+                requestors = instructors
```

Comment:
can we indent after the `with`? Such as 

```suggestion
                  requestors = instructors
```

Also, are we changing from zero-based indexing to one-based indexing of instructions? This would be another breaking change.

Reply:
agree